GliaWidgets/SecureConversations/SecureConversations.AcitivityIndicator.swift

```diff
@@ -0,0 +1,202 @@
+import UIKit
```

Comment:
File has a typo, should be `ActivityIndicator`

Reply:
I'm good at typos e26720332af25144df7c240c08665f8d732e5941.